python_modules/dagster/dagster/scheduler/scheduler.py

```diff
@@ -288,17 +309,14 @@ def _schedule_runs_at_time(
                     f"Run {run.run_id} already created for this execution of {external_schedule.name}"
                 )
         else:
-            run, errors = _create_scheduler_run(
+            run = _create_scheduler_run(
```

Comment:
this has different behavior than before, right?

before, it would attempt to launch as much as possible (catching errors early).  Now it bubbles up to the tick context and then we rely on the retry mechanism?

Reply:
kind of. Before, if there was an error while generating the execution plan, it would still create a run, but immediately mark it as failed.

I claim that this was a much more useful thing back when the only 'scheduler' we had was running on the crontab, and we didn't have the schedule timeline view, etc. - so there was no other way to figure out that a tick failed like this than going to the runs page.

Now, if there's an error while fetching your execution plan, you still have a place to go - and the behavior  that this is replacing is inconsistent with both sensors and launching an ad-hoc run (sensors just raise the error)